lib/spack/spack/test/cmd/list.py

```diff
@@ -45,6 +45,14 @@ def test_list_format_name_only():
 
 
 @pytest.mark.maybeslow
+def test_list_format_version_json():
```

Comment:
We moved the marker from `test_list_format_html` to `test_list_format_version_json`?

Reply:
urgh, unintentional. fixed!